client/data/launchpad-navigator/use-launchpad-navigator.ts

```diff
@@ -1,22 +1,31 @@
 import { useQuery } from '@tanstack/react-query';
-import apiFetch, { APIFetchOptions } from '@wordpress/api-fetch';
+import apiFetch, { type APIFetchOptions } from '@wordpress/api-fetch';
 import wpcomRequest, { canAccessWpcomApis } from 'wpcom-proxy-request';
 
 interface AvailableChecklist {
 	slug: string;
 	title: string;
 }
 
+interface AvailableChecklists {
+	[ checklist_slug: string ]: AvailableChecklist;
+}
```

Comment:
Non-blocking thought, which does use a stricter type instead of an interface.
It may be cleaner/simpler to use the built-in `Record` type here, but I don't know that it helps much.
```suggestion
type AvailableChecklists = Record< string, AvailableChecklist >;
```
We could also declare `type ChecklistSlug = string;` but I don't think that buys us much.